templates/launcher/application.ejs

```diff
@@ -137,6 +137,14 @@ function initializeGame (cc, settings, findCanvas) {
                 cc.Layers.addLayer(layer.name, layer.bit);
             });
         }
+        if (settings.sortingLayersUsed) {
+            cc.sortingManager.initLayerIDFromSettings(settings.sortingLayersUsed);
+        }
+        if (settings.sortingLayers) {
+            settings.sortingLayers.forEach((sortingLayer) => {
```

Comment:
hack for now, I'm working on restructure the launch process，and after that `game.init` will be like this 
![image](https://user-images.githubusercontent.com/15109777/168421797-49414b1f-509f-499e-b6a9-8bf93f74939d.png)
should adjust it after refactoring

Reply:
I‘ll update when it change